graphdatascience/tests/integration/test_database_ops.py

```diff
@@ -10,6 +12,8 @@
 from graphdatascience.version import __version__
 
 GRAPH_NAME = "g"
+RETRY_TRIALS = 10
+SLEEP_TIME = 0.5
```

Comment:
It is possible to use blocking variants of Neo4j system commands. Usually with the syntax `WAIT` (and `NO WAIT`).

So doing `CREATE DATABASE foo WAIT` is a better alternative here.